apis/scheduling/config/types.go

```diff
@@ -123,3 +123,13 @@ type ReservationArgs struct {
 	// EnablePreemption indicates whether to enable preemption for reservations.
 	EnablePreemption *bool `json:"enablePreemption,omitempty"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// GangSchedulingArgs defines the parameters for GangScheduling plugin.
+type GangSchedulingArgs struct {
+	metav1.TypeMeta
+
+	//DefaultTimeOutSeconds is the default gang's waiting time in Permit stage.
+	DefaultTimeOutSeconds *int64 `json:"defaultTimeOutSeconds,omitempty"`
```

Comment:
It is recommended to use *metav1.Duration, which is easier to use

Reply:
right,solved